# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/swappy/swappy-git-version.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Tarball has no .git, so rev-parse fails and check:true aborts the
configure step before the returncode guard. Tolerate the failure,
the guard already falls back to the plain project version.

Signed-off-by: Dennie Mahne <dennie.mahne@gmail.com>

--- swappy-1.8.0/meson.build.orig	2025-08-27 21:34:53.000000000 +0200
+++ swappy-1.8.0/meson.build	2026-09-29 12:31:26.552177510 +0200
@@ -14,8 +14,8 @@
 version = '"@0@"'.format(meson.project_version())
 git = find_program('git', native: true, required: false)
 if git.found()
-	git_commit = run_command([git, 'rev-parse', '--short', 'HEAD'], check:true)
-	git_branch = run_command([git, 'rev-parse', '--abbrev-ref', 'HEAD'], check:true)
+	git_commit = run_command([git, 'rev-parse', '--short', 'HEAD'], check:false)
+	git_branch = run_command([git, 'rev-parse', '--abbrev-ref', 'HEAD'], check:false)
 	if git_commit.returncode() == 0 and git_branch.returncode() == 0
 		version = '"@0@-@1@ (" __DATE__ ", branch \'@2@\')"'.format(
 			meson.project_version(),
